include/tvm/relay/op.h

```diff
@@ -610,6 +610,19 @@ inline bool IsPrimitiveOp(const Expr& expr) {
   return op != nullptr && op->IsPrimitiveOp();
 }
 
+/*!
+ * \brief Check if an op is the one with the provided name.
+ *
+ * \param ref The op to be checked.
+ * \param name The given operator name.
+ *
+ * \return True if the op is the same with the given name. Otherwise, false.
+ */
+inline bool IsOp(const NodeRef& ref, const std::string& name) {
+  CHECK(ref.defined());
+  return ref->IsInstance<OpNode>() && Downcast<Op>(ref) == Op::Get(name);
```

Comment:
We may need to add a checker to make sure `Op::Get()` retuerns a valid op in case the given op name string has a typo. The checker can also help debugging; otherwise it's hard to find the problems if `IsOp` returns false due to typos.

Reply:
yes, it is here:

https://github.com/apache/incubator-tvm/blob/e6d9f89c1ce9c45d0c65dce582de440c2fc7b8ba/src/relay/ir/op.cc#L62-L65

Reply:
Oops. I missed looking into the `Get` function. Feel free to resolve this.